### DISCUSSION TITLE
No Cancel trial Button

### DISCUSSION THREAD

**Discussion by MotaOcimar** [community/community#46301] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Unlike what is shown in the documentation, my settings page does not show the option to cancel the Copilot Trial.

Documentation image:
![doc image](https://docs.github.com/assets/cb-103284/images/help/copilot/copilot-cancel-trial.png)

How it appears on my settings page:
![image](https://user-images.githubusercontent.com/29007115/216660410-5a33f423-c0ca-4d6c-b174-d807e896a90b.png)

I want to cancel the trial and avoid payments. But I am unable to do this.

**Comment by chris-c-thomas** [comment]
Oh wow. Try to logout and login again or another browser? It's showing for me.

**Reply by MotaOcimar** [reply]
I've tried Chrome, Firefox and Safari. None of them showed me the cancel button 😥

**Comment by chris-c-thomas** [comment]
Hmm, actually do you even have a card on file? I see you're not a Pro member. That might actually be the reason. If you don't they won't actually charge you on account of not having anything to charge.

**Comment by chris-c-thomas** [comment, marked as answer]
Otherwise I'd most certainly send support a message at https://support.github.com/request or tweet them at https://twitter.com/GitHubHelp

**Comment by MotaOcimar** [comment]
Hi @chris-c-thomas!

Thanks for the suggestions! The problem was solved and I almost forgot to reply you. But I talked to support and they help me very quickly!

**Reply by chris-c-thomas** [reply]
Glad to hear it!
